LayoutTests/platform/mac-wk2/imported/w3c/web-platform-tests/html/rendering/widgets/field-sizing-input-file.optional-expected.txt

```diff
@@ -0,0 +1,9 @@
+
+
+PASS file: Empty value
+FAIL file: Auto width with a short filename assert_less_than: expected a number less than 240 but got 267
```

Comment:
Similar to WK1 but on WK2 it sets it to "com.apple.WebKit.WebContent.xpc" see https://bugs.webkit.org/show_bug.cgi?id=293584